using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -89,7 +101,11 @@ This API synchronizes the provided customer identifier type key and value with t
          authentication: (ADBMobileVisitorAuthenticationState) authenticationState;
 ```
 
-#### **Examples**
+o  *identifierType* (String) containing identifier type, and the *identifier* (String) containing identifier value; should not be null or empty. If the identifierTtype and identifier do not have valid string values \(non null or empty\), the identifier is ignored by the Identity extension.
```

Comment:
- Please remove the "o" and keep the asterisks for both parameters.

- Please reword the 1st sentence so that it reads:
"The identifierType* (String) contains the identifier type, and the *identifier* (String) contains the identifier value. If the `identifierType` and identifier have a non-null value or are empty, the identifier is ignored by the Identity extension.